Formula/pip-tools.rb

```diff
@@ -45,9 +45,14 @@ class PipTools < Formula
     sha256 "2b020ecf7d21b687f219b71ecad3631f644a47f01403fa1d1036b0c6416d70fb"
   end
 
+  resource "tomli" do
+    url "https://files.pythonhosted.org/packages/c0/3f/d7af728f075fb08564c5949a9c95e44352e23dee646869fa104a3b2060a3/tomli-2.0.1.tar.gz"
+    sha256 "de526c12914f0c550d15924c62d72abc48d6fe7364aa87328337a31007fe8a4f"
+  end
```

Comment:
Can you reinstall `pipgrip`? This is probably using Python 3.10 version of `pipgrip` as `tomli` shouldn't exist when getting dependencies on Python 3.11:
```console
❯ pipgrip --tree pip-tools
pip-tools (6.10.0)
├── build (0.9.0)
│   ├── packaging>=19.0 (21.3)
│   │   └── pyparsing!=3.0.5,>=2.0.2 (3.0.9)
│   └── pep517>=0.9.1 (0.13.0)
├── click>=7 (8.1.3)
├── pip>=21.2 (22.3.1)
├── setuptools (65.5.1)
└── wheel (0.38.4)
```

```console
❯ /usr/local/bin/python3.10 -m venv venv
❯ source venv/bin/activate
❯ pip install pipgrip -qqq
❯ venv/bin/pipgrip --tree pip-tools
pip-tools (6.10.0)
├── build (0.9.0)
│   ├── packaging>=19.0 (21.3)
│   │   └── pyparsing!=3.0.5,>=2.0.2 (3.0.9)
│   ├── pep517>=0.9.1 (0.13.0)
│   │   └── tomli>=1.1.0 (2.0.1)
│   └── tomli>=1.0.0 (2.0.1)
├── click>=7 (8.1.3)
├── pip>=21.2 (22.3.1)
├── setuptools (65.5.1)
└── wheel (0.38.4)
```

Reply:
Note that does mean that after reinstallation, any `brew bump` PRs for Python 3.10 formulae will use the Python 3.11 dependency tree.

Reply:
Done. It probably was moved to python 3.11 silently (no revision bump nor version upgrade) and hence was still on version 3.10. Do I get rid of the block? Needs to rerun something just in case?

Reply:
Deleting block is fine. If you want, you can do `brew update-python-resources pip-tools` which will re-run same command from bump.